stripe-ui-core/res/drawable/stripe_ic_card_visa.xml

```diff
@@ -0,0 +1,17 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="56dp"
+    android:height="40dp"
+    android:viewportWidth="56"
+    android:viewportHeight="40">
+    <path
+        android:pathData="M39.5067,13.748C39.5891,13.2735 39.3216,12.8078 38.8703,12.6398C38.0406,12.331 36.7536,12 35.1712,12C33.1712,12 31.383,12.499 30.0634,13.4375C28.727,14.3879 27.8809,15.7915 27.8692,17.473C27.8497,18.8144 28.5054,19.8241 29.2866,20.5439C30.0521,21.2492 30.9968,21.7346 31.7256,22.0691C32.4376,22.3959 32.8078,22.6206 32.995,22.7841C33.0263,22.8114 33.048,22.8333 33.0626,22.8495C33.0384,22.8742 32.989,22.9151 32.8983,22.9617C32.654,23.0872 32.2651,23.1739 31.8359,23.1739C30.5026,23.1739 29.8342,22.9946 28.7247,22.5336L28.2424,22.3213C27.9606,22.1972 27.6374,22.2099 27.3662,22.3556C27.095,22.5013 26.906,22.7638 26.8539,23.0672L26.3312,26.1094C26.2528,26.5658 26.498,27.0163 26.9238,27.1983C27.9517,27.6376 29.7036,27.9846 31.4739,28H31.4826C33.5616,28 35.3854,27.5247 36.73,26.5841C36.6735,26.8423 36.7211,27.1157 36.8679,27.3421C37.0523,27.6263 37.3681,27.7978 37.7069,27.7978H41.6261C42.0396,27.7978 42.4103,27.5434 42.5591,27.1577L43.0963,25.7649H46.4026L46.6743,27.0102C46.7742,27.4699 47.1811,27.7978 47.6515,27.7978H51.11C51.4133,27.7978 51.7002,27.6601 51.89,27.4235C52.0798,27.1869 52.152,26.8769 52.0861,26.5808L49.0727,13.0286C48.971,12.5711 48.5652,12.2457 48.0966,12.2457H45.2074C44.683,12.2457 44.1089,12.3153 43.5793,12.5911C43.0106,12.8873 42.5939,13.3873 42.3388,13.9696L38.9235,21.6196C38.8003,20.9048 38.4887,20.2496 37.9912,19.6631C37.3442,18.9004 36.4138,18.2885 35.2677,17.773C34.5999,17.4559 34.1637,17.2357 33.8891,17.0381L33.8647,17.0203C34.0335,16.9361 34.3548,16.8406 34.8932,16.8406L34.9171,16.8403C35.95,16.8157 36.7009,17.0441 37.292,17.2831L37.6216,17.4235C37.9026,17.5432 38.2229,17.5279 38.4912,17.3819C38.7596,17.236 38.9465,16.9754 38.9987,16.6745L39.5067,13.748Z"
```

Comment:
Ideally, I would like to avoid adding a public resource in this library. The only way I've found to be able to reference a test-only resource is by having it in `androidTest`.